ci/gpu/build.sh

```diff
@@ -43,10 +43,10 @@ nvidia-smi
 logger "Activate conda env..."
 source activate gdf
 conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
-              "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.16" "double-conversion" \
+              "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.17.4" "double-conversion" \
```

Comment:
What requires such a new constraint of the numpy version here?

Reply:
Reverted this back to it's original state.